bundles/ch.itmed.fop.printing/res/xsl/PatientLabel.xsl

```diff
@@ -79,7 +80,13 @@
 			<xsl:if test="/Page/Patient/MobilePhone/text()">
 				&#160;Mobil:&#160;
 				<xsl:value-of select="MobilePhone" />
+
 			</xsl:if>
 		</fo:block>
+		<fo:block>
+			Druckdatum:
+			<xsl:value-of select="PrintingDate" />
+			&#160;
+			</fo:block>

```

Comment:
Wir das mit folgender Änderung ersetzt, sind die anderen Änderungen nicht nötig. Da im Page Element bereits ein currentDate Attribut abgefüllt wird.


		<fo:block>
			Druckdatum:
			<xsl:value-of select="/Page/@currentDate" />
			&#160;
		</fo:block>